build/DownloadedPackageVersions.targets

```diff
@@ -0,0 +1,11 @@
+<Project ToolsVersion="15.0">
+  <Target Name="DownloadPackageVersionsProps">
+    <PropertyGroup>
+      <PB_AssetRootAccessTokenSuffixParam Condition=" '$(PB_AssetRootAccessTokenSuffix)' != '' ">?$(PB_AssetRootAccessTokenSuffix)</PB_AssetRootAccessTokenSuffixParam>
```

Comment:
Are these access tokens going to be different? Why not use the same one we are using for the blob storage?

Reply:
Fixed...